client/my-sites/media-library/content.jsx

```diff
@@ -449,7 +453,7 @@ export class MediaLibraryContent extends Component {
 					selectedItems={ this.props.selectedItems }
 					sticky={ ! this.props.scrollable }
 					hasAttribution={ 'pexels' === this.props.source }
```

Comment:
Unless Openverse queries are restircted to licenses or marks that do not require attribution, this should be updated to have Openverse images with attribution.

Reply:
From what I could gather, `hasAttribution` displays an attribution to Pexels itself for providing these images (Photos provided by Pexels):

<img width="679" alt="Screen Shot 2022-01-28 at 9 18 53 AM" src="https://user-images.githubusercontent.com/1398304/151592499-43eebe9c-bcd0-4e3c-b694-20213bd3bffb.png">

Attribution for the pictures that get inserted into post content are generated server-side, when parsing images from search results, and saved in caption.

Reply:
Aha, gotcha! Should there be an equivalent Openverse attribution then?

Reply:
I don't know, [it seems to be Pexels-specific](https://github.com/Automattic/wp-calypso/blob/627e76291fbac621945bcaf0ab6d6b0de24cd05e/client/my-sites/media-library/external-media-header.jsx#L104-L112), which makes me wonder if it was a contractual thing

Reply:
Ahh, I suppose maybe they require attribution even on the collection of images displayed in the search results, not just on the individual embed itself? 🤔